### ISSUE TITLE
[Accessibility] Tooltip content is not accessible to screen readers.

### ISSUE THREAD

**Issue by puneet-ahuja** [vega/vega#3879]
Tooltips in the charts are not accessible to the screen readers.

Steps to replicate:

Open the [Codesandbox Link](https://codesandbox.io/p/sandbox/tooltip-not-accessible-with-keyboard-yh3ldj?file=%2Findex.html%3A63%2C8).

The above link opens an interactive multi-line chart.
The content in the tooltip is not accessible to the screen readers.

**Comment by domoritz** [comment]
The current behavior somewhat by design. Tooltips are designed for mouse-interactions. The details of the marks should be added to the marks as aria properties so that the tooltips are not needed at all. See https://vega.github.io/vega/docs/marks/#accessibility-properties-51. Vega-Lite does that by default already IIRC. 

However, I see that that is not aways the case and that we could do a better job with documentation and maybe exposing the tooltip values by default to a screen reader. 

I don't know how to act on this current issue since Vega is fairly low-level and so it's more up to the developer how they want to use the library. We could definitely improve docs and also the default behavior in Vega-Lite but those would need separate and more specific issues.
